vagrant/prepare-disk.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# To start again and wipe out the disk, in this order:
+#
+#   wipefs -a /dev/sdb1
+#   fdisk /dev/sdb ; d; w
+
+DATA_MOUNT=/mnt/data
+
+mkdir -p $DATA_MOUNT
+
+if [ ! -b /dev/sdb1 ]; then
+    echo "Adding partition table to /dev/sdb"
+    echo ';' | sfdisk /dev/sdb
```

Comment:
What does this line do? I mean, apparently it adds a partition table, but why are you piping `;` to sfdisk and how does this work?